docs/index.md

```diff
@@ -59,7 +59,7 @@ Steve Nash (snash) - James Tokarsky (thejat) - RobinBennett - Ian (ianj001) - Ge
 
 ## About V1
 
-V1 Engineering Inc all started with the [MPCNC platform](mpcnc/index.md), from there it kept expanding.
+V1 Engineering Inc all started with the [MPCNC platform](mpcnc/burly/index.md), from there it kept expanding.

```

Comment:
Maybe you've already decided this, but we should be clear as to which version we should mainly be pointing to. I'm assuming sometime soon you will want to point this to primo and not burly.